app.js

```diff
@@ -0,0 +1,73 @@
+//Declaring global variables
+let topics = ['berlin', 'paris', 'new york city', 'london', 'moscow', 'beijing', 'los angeles', 'lagos', 'delhi', 'shanghai', 'mammoth lakes']
+
+topics.forEach(topic => {
+    $('#cityButtons').append(`
+    <button type="button" class="btn btn-outline-primary" data-city='${topic}'>${topic}</button>
+    `)
+});
+
+//Adds new city inputs to array and appends on page
+function submitButton () {
+    event.preventDefault();
+    let newCity = $('#city-input').val();
+    topics.push(newCity)
+    $('#cityButtons').append(`
+    <button type="button" class="btn btn-outline-primary" data-city='${newCity}'>${newCity}</button>`)
+    $('#city-input').val('')
+    console.log(newCity)
```

Comment:
make it a habit of removing console logs once you deploy a project so the user can't see your consoles.